helm/cluster-eks/templates/coredns-helmrelease.yaml

```diff
@@ -41,3 +41,6 @@ spec:
           clusterIPRange: {{ .Values.connectivity.network.serviceCidr | quote }}
         DNS:
           IP: {{ include "clusterDNS" $ | quote }}
+    mastersInstance:
+      nodeSelector:
+        "node-role.kubernetes.io/worker": '""'
```

Comment:
there are no control-plane machines on EKS so we set the  node selector same for both deployments to avoid alerting or errors and in order to be able to run the same app